### PR TITLE
Dev navbar

### PR DESCRIPTION
**Pull request by Temitopefa**
Developer Navbar created for the dashboard.

### CODE REVIEW

File: src/components/DevNavbar.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+import { makeStyles } from "@mui/styles";
+import { Link } from 'react-router-dom';
+
+import Menus from '../components/Menus'
+
```
Comment (pablo-clueless) [inline src/components/DevNavbar.tsx]: Reduce the spaces here. 1 line is enough space

File: src/components/DevNavbar.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+import { makeStyles } from "@mui/styles";
+import { Link } from 'react-router-dom';
+
+import Menus from '../components/Menus'
```
Comment (pablo-clueless) [inline src/components/DevNavbar.tsx]: - Named import?? `import { Menus} from "../components"
- Double quotes too and semicolons.

File: src/components/Menus.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { makeStyles } from "@mui/styles";
+
+import Button from '@mui/material/Button';
```
Comment (pablo-clueless) [inline src/components/Menus.tsx]: Import all these components on same line

```
import { Avatar, Button, Fade, Menu, MenuItems} from "@mui/material";
```

File: src/components/Menus.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+  return (
+     <div className={className}>
+            <Button 
+                sx={{ width:'450px' }}
+                id="fade-button"
+                aria-controls={open ? 'fade-menu' : undefined}
+                aria-haspopup="true"
+                aria-expanded={open ? 'true' : undefined}
+                onClick={handleClick}
```
Comment (pablo-clueless) [inline src/components/Menus.tsx]: Don't spread stuff like this, it makes the file larger. You can do it while working but put them back on the same line when pushing.

File: src/components/DevNavbar.tsx (hunk 2 above)
Comment (Temitopefa) [reply src/components/DevNavbar.tsx]: Done

File: src/components/Menus.tsx (hunk 3 above)
Comment (Temitopefa) [reply src/components/Menus.tsx]: Done

File: src/components/Menus.tsx (hunk 4 above)
Comment (Temitopefa) [reply src/components/Menus.tsx]: Done

File: src/components/DevNavbar.tsx (hunk 1 above)
Comment (Temitopefa) [reply src/components/DevNavbar.tsx]: Done

**Review by pablo-clueless — APPROVED** [review]

**Review by taiwoAkindele — APPROVED** [review]

**Review by Vinyl-Davyl — APPROVED** [review]

**Pull request merged**